core/audits/seo/crawlable-anchors.js

```diff
@@ -44,9 +44,12 @@ class CrawlableAnchors extends Audit {
       name = '',
       role = '',
       id,
+      href,
     }) => {
       rawHref = rawHref.replace( /\s/g, '');
       name = name.trim();
+
+      if (role === ' ') return true;
```

Comment:
What's this change for?

Reply:
this is for the test case in https://github.com/GoogleChrome/lighthouse/pull/15406/files#diff-0f0c5dba54690c77d2d018d883926a9ac5594bef8fb4162fbe8097bc3ee6cd56R82. This would essentially only fail because of the`if (rawHref === '') return true;` condition.

Reply:
I just tried removing this case and all the tests passed, it should be handled by the `role = role.trim()` case below